src/Security/Member.php

```diff
@@ -1044,7 +924,7 @@ public function onAfterWrite()
 
         Permission::reset();
 
-        if ($this->isChanged('Password')) {
+        if ($this->isChanged('Password') && static::config()->get('password_logging_enabled')) {
```

Comment:
The idea comes from a previous job, where on two accounts, the client explicitly asked for no password history (not sure if "security risk" at all, but that was their argument)